src/Client.js

```diff
@@ -1037,13 +1037,13 @@ class Client extends EventEmitter {
      * @returns {Promise<Object|null>}
      */
     async getNumberId(number) {
-        if (!number.endsWith('@c.us')) {
-            number += '@c.us';
-        }
+        //if (!number.endsWith('@c.us')) {
```

Comment:
remove comments them we are good to commit

Reply:
Ok